tests/test_accounts_service.py

```diff
@@ -0,0 +1,239 @@
+                         '/org/freedesktop/Accounts/User2001')
+        # FIXME: This requires AccountsService 8f07b100 # pylint: disable=fixme
+        # self.assertEqual(user.get_password_expiration_policy(),
+        #                  (1000, 1615832413, 0, 0, 0, 0))
+        self.assertEqual(user.get_password_hint(), 'Remember it, come on!')
+        self.assertEqual(user.get_password_mode(),
+                         AccountsService.UserPasswordMode.REGULAR)
+        self.assertIsNone(user.get_primary_session_id())
+        self.assertEqual(user.get_real_name(), 'I Love Pizza')
+        self.assertFalse(user.get_saved())
+        self.assertEqual(user.get_session(), 'mock-session')
+        self.assertEqual(user.get_session_type(), 'wayland')
+        self.assertEqual(user.get_shell(), '/bin/zsh')
+        self.assertEqual(user.get_uid(), 2001)
+        self.assertEqual(user.get_user_name(), 'pizza')
+        self.assertEqual(user.get_x_session(), 'mock-xsession')
+        self.assertTrue(user.is_loaded())
+        self.assertTrue(user.is_local_account())
+        self.assertFalse(user.is_logged_in())
+        self.assertFalse(user.is_logged_in_anywhere())
+        self.assertFalse(user.is_nonexistent())
+        self.assertFalse(user.is_system_account())
+
+        other = self.p_manager.create_user(
+            'schiacciata', 'I Love Schiacciata too',
+            AccountsService.UserAccountType.STANDARD)
+
+        while not self.p_manager.props.has_multiple_users:
+            self.ctx.iteration(True)
+
+        self.assertTrue(self.p_manager.props.has_multiple_users)
+        self.assertIn(other, self.p_manager.list_users())
+        self.assertEqual(other.get_uid(), 2002)
+        self.assertEqual(other.get_user_name(), 'schiacciata')
+        self.assertEqual(other.get_real_name(), 'I Love Schiacciata too')
+        self.assertEqual(other.get_account_type(),
+                         AccountsService.UserAccountType.STANDARD)
+
+    def test_set_user_properties(self):
+        # pylint: disable=too-many-statements
+        user = self.p_manager.create_user(
+            'test-user', 'I am a Test user',
+            AccountsService.UserAccountType.STANDARD)
+
+        user.set_account_type(
+            AccountsService.UserAccountType.ADMINISTRATOR)
+        self.wait_changed(user)
+        self.assertEqual(user.get_account_type(),
+                         AccountsService.UserAccountType.ADMINISTRATOR)
+
+        user.set_account_type(
+            AccountsService.UserAccountType.STANDARD)
+        self.wait_changed(user)
+        self.assertEqual(user.get_account_type(),
+                         AccountsService.UserAccountType.STANDARD)
+
+        user.set_automatic_login(True)
+        self.wait_changed(user)
+        self.assertTrue(user.get_automatic_login())
+
+        user.set_automatic_login(False)
+        self.wait_changed(user)
+        self.assertFalse(user.get_automatic_login())
+
+        user.set_email('Test Email')
```

Comment:
let's please use a validly formed email, both to make this more readable and also prevent failure if the daemon or the library ever starts to check this.